docs/admin.rst

```diff
@@ -12,9 +12,57 @@ The SecureDrop architecture contains multiple machines and hardened servers.
 While many of the installation and maintenance tasks have been automated, a
 skilled Linux admin is required to responsibly run the system.
 
-This section outlines the tasks the admin is responsible for in order to
-ensure that their SecureDrop instance continues to be a safe place for sources to
-talk to journalists.
+Responsibilities of SecureDrop administrators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+As a SecureDrop administrator, it is your responsibility to:
+
+* :ref:`manage users <manage_users>`
+* :ref:`manage the system configuration <manage_config>`
+* :ref:`ensure that servers and workstations are kept up-to-date <manage_updates>`
+* :ref:`monitor OSSEC alerts <monitoring_ossec>`
+* :ref:`monitor the SecureDrop team's release and security-related
```

Comment:
and physical security

Reply:
added in 3c6dd8f5e6106c4044ce90451c36efe96e18b56a